### PR TITLE
Add direct conversion from Ydb::Value tuple to Cell in rpc_read_table

### PR DESCRIPTION
**Pull request by lex007in**
### Changelog entry 

Add direct conversion from Ydb::Value tuple to Cell in rpc_read_table

### Changelog category 

* Improvement

### Additional information

...

### CODE REVIEW

File: ydb/core/tx/schemeshard/ut_column_build/ut_column_build.cpp [hunk 1]
```diff
@@ -215,7 +216,7 @@ Y_UNIT_TEST_SUITE(ColumnBuildTest) {
 
         Ydb::TypedValue defaultValue;
         defaultValue.mutable_type()->set_type_id(Ydb::Type::UINT64);
-        defaultValue.mutable_value()->set_text_value("1111");
+        defaultValue.mutable_value()->set_uint64_value(1111); // TODO: check invalid value
```
Comment (lex007in) [inline ydb/core/tx/schemeshard/ut_column_build/ut_column_build.cpp:219]: It looks like it needs to be validated somewhere here: https://github.com/ydb-platform/ydb/blob/main/ydb/core/tx/schemeshard/schemeshard_build_index__create.cpp#L161

**Review by gridnevvvit — APPROVED** [review]

File: ydb/core/tx/schemeshard/ut_column_build/ut_column_build.cpp (hunk 1 above)
Comment (gridnevvvit) [reply ydb/core/tx/schemeshard/ut_column_build/ut_column_build.cpp:219]: okay from my side here. I'll check this unit a bit later.

File: ydb/services/ydb/ut/ya.make [hunk 2]
```diff
@@ -38,7 +38,7 @@ PEERDIR(
     library/cpp/regex/pcre
     library/cpp/svnversion
     ydb/core/kqp/ut/common
-    ydb/core/testlib/default
+    ydb/core/testlib/pg
```
Comment (MBkkt) [inline ydb/services/ydb/ut/ya.make:41]: Is it intentionally?

File: ydb/core/ydb_convert/ydb_convert.h [hunk 3]
```diff
@@ -64,4 +65,20 @@ bool FillACL(NKikimrSchemeOp::TModifyScheme& out,
...
+    NScheme::TTypeInfo TypeInfo;
+    TString TypeMod;
+    bool IsNotNull;
+};
+
+bool CellsFromTuple(
+    const Ydb::Type* tupleType, // nullptr if no type verification needed
+    const Ydb::Value& tupleValue,
+    const TConstArrayRef<TConversionTypeInfo>& types,
+    bool allowCastFromString,
```
Comment (MBkkt) [inline ydb/core/ydb_convert/ydb_convert.h:78]: is it about implicit cast from string literal in postgres?

File: ydb/core/ydb_convert/ydb_convert.cpp [hunk 4]
```diff
@@ -1464,4 +1492,96 @@ void FillOwner(NKikimrScheme::TEvModifySchemeTransaction& out,
...
+                    TString& errStr,
+                    TMemoryPool& valueDataPool) {
+    if (tupleType) {
+        Ydb::Type::TypeCase typeCase = tupleType->type_case();
+        CHECK_OR_RETURN_ERROR(typeCase == Ydb::Type::kTupleType ||
+                              (typeCase == Ydb::Type::TYPE_NOT_SET && tupleType->tuple_type().elementsSize() == 0), "Must be a tuple");
+        CHECK_OR_RETURN_ERROR(tupleType->tuple_type().elementsSize() <= types.size(),
+            "Tuple size " + ToString(tupleType->tuple_type().elementsSize()) + " is greater that expected size " + ToString(types.size()));
+
+        for (size_t i = 0; i < tupleType->tuple_type().elementsSize(); ++i) {
```
Comment (MBkkt) [inline ydb/core/ydb_convert/ydb_convert.cpp:1517]: I thought we use snake_case for public api protos

File: ydb/core/ydb_convert/ut/ya.make [hunk 5]
```diff
@@ -14,7 +14,7 @@ SRCS(
 
 PEERDIR(
     library/cpp/testing/unittest
-    ydb/core/testlib/default
+    ydb/core/testlib/pg
```
Comment (MBkkt) [inline ydb/core/ydb_convert/ut/ya.make:17]: is it intentional?

File: ydb/services/ydb/ut/ya.make (hunk 2 above)
Comment (lex007in) [reply ydb/services/ydb/ut/ya.make:41]: Yes, testlib/default has dummy implementation for pg types interfaces and catalog

File: ydb/core/ydb_convert/ydb_convert.h (hunk 3 above)
Comment (lex007in) [reply ydb/core/ydb_convert/ydb_convert.h:78]: Yes, but no. It's about implicit casts for native YDB types only. It's from original impl of CellsFromTuple for NKikimrMiniKQL::TValue: https://github.com/ydb-platform/ydb/blob/main/ydb/core/engine/mkql_proto.cpp#L17

File: ydb/core/ydb_convert/ydb_convert.cpp (hunk 4 above)
Comment (lex007in) [reply ydb/core/ydb_convert/ydb_convert.cpp:1517]: It's different in different places) But in this file mostly snake case is used, so I'l fix that.

File: ydb/core/ydb_convert/ut/ya.make (hunk 5 above)
Comment (lex007in) [reply ydb/core/ydb_convert/ut/ya.make:17]: Yes, testlib/default has dummy implementation for pg types interfaces and catalog

File: ydb/core/ydb_convert/ydb_convert.cpp (hunk 4 above)
Comment (MBkkt) [reply ydb/core/ydb_convert/ydb_convert.cpp:1517]: ok

**Review by MBkkt — APPROVED** [review]

**Pull request merged**